scripts/deploy-backend.sh

```diff
@@ -67,66 +71,124 @@ set_cf_envs()
+    yq eval -i ".applications[0].services[1] = \"tdp-staticfiles-${env}\"" ./tdrs-backend/manifest.celery.yml
+    yq eval -i ".applications[0].services[2] = \"tdp-datafiles-${env}\"" ./tdrs-backend/manifest.celery.yml
+    yq eval -i ".applications[0].services[3] = \"tdp-redis-${env}\"" ./tdrs-backend/manifest.celery.yml
 }
 
 update_backend()
 {
+    APP=$1
+    STRATEGY=$2
+
+    echo "Deploying $APP with strategy $STRATEGY"
+
     cd tdrs-backend || exit
-    cf unset-env "$CGAPPNAME_BACKEND" "AV_SCAN_URL"
 
-    if [ "$CF_SPACE" = "tanf-prod" ]; then
-      cf set-env "$CGAPPNAME_BACKEND" AV_SCAN_URL "http://tanf-prod-clamav-rest.apps.internal:9000/scan"
+    if [ "$APP" = "$CGAPPNAME_BACKEND" ] ; then
+      MANIFEST="manifest.buildpack.yml"
+    elif [ "$APP" = "$CGAPPNAME_CELERY" ] ; then
+      MANIFEST="manifest.celery.yml"
     else
-      # Add environment varilables for clamav
-      cf set-env "$CGAPPNAME_BACKEND" AV_SCAN_URL "http://tdp-clamav-nginx-$env.apps.internal:9000/scan"
-
-      # Add variable for dev/staging apps to know their DB name. Prod uses default AWS name.
-      cf unset-env "$CGAPPNAME_BACKEND" "APP_DB_NAME"
-      cf set-env "$CGAPPNAME_BACKEND" "APP_DB_NAME" "tdp_db_$backend_app_name"
+      echo "Unknown app"
+      exit
     fi
 
-    if [ "$1" = "rolling" ] ; then
-        set_cf_envs
+    if [ "$STRATEGY" = "rolling" ] ; then
+        set_cf_envs "$APP"
         # Do a zero downtime deploy.  This requires enough memory for
         # two apps to exist in the org/space at one time.
-        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml -t 180 --strategy rolling || exit 1
+        cf push "$APP" --no-route -f "$MANIFEST" -t 180 --strategy rolling || exit 1
     else
-        cf push "$CGAPPNAME_BACKEND" --no-route -f manifest.buildpack.yml -t 180
+        cf push "$APP" --no-route -f "$MANIFEST" -t 180
+
         # set up JWT key if needed
-        if cf e "$CGAPPNAME_BACKEND" | grep -q JWT_KEY ; then
+        if cf e "$APP" | grep -q JWT_KEY ; then
             echo jwt cert already created
         else
-            generate_jwt_cert
+            generate_jwt_cert "$APP"
         fi
+
+        set_cf_envs "$APP"
+
+        cf unset-env "$APP" "AV_SCAN_URL"
+
+        if [ "$CF_SPACE" = "tanf-prod" ]; then
+          cf set-env "$APP" AV_SCAN_URL "http://tanf-prod-clamav-rest.apps.internal:9000/scan"
+        else
+          # Add environment varilables for clamav
+          cf set-env "$APP" AV_SCAN_URL "http://tdp-clamav-nginx-$env.apps.internal:9000/scan"
+
+          # Add variable for dev/staging apps to know their DB name. Prod uses default AWS name.
+          cf unset-env "$APP" "APP_DB_NAME"
+          cf set-env "$APP" "APP_DB_NAME" "tdp_db_$backend_app_name"
+        fi
+
+        cf set-env "$APP" CGAPPNAME_BACKEND "$CGAPPNAME_BACKEND"
+
+        cf restage "$APP"
     fi
 
-    set_cf_envs
+    cd ..
+}
 
+update_backend_network()
```

Comment:
need a network for celery as well to enable prometheus scrape config to work (`plg/deploy.sh`)